src/sentry/snuba/discover.py

```diff
@@ -831,7 +839,9 @@ def get_performance_facets(
             )
             results.extend(
                 [
-                    FacetResult(r["tags_key"], r["tags_value"], int(r["aggregate"]))
+                    PerformanceFacetResult(
+                        r["tags_key"], r["tags_value"], int(r["aggregate"]), int(r["count"])
```

Comment:
Was the aggregate intentionally casted to an `int` here?

```suggestion
                        r["tags_key"], r["tags_value"], float(r["aggregate"]), int(r["count"])
```